core/templates/dev/head/language_footer.html

```diff
@@ -1,6 +1,6 @@
 <div class="oppia-language-footer" ng-controller="I18nFooter">
   <span translate="I18N_LANGUAGE_FOOTER_VIEW_IN"></span>
-  <span ng-repeat="(langCode, langName) in supportedSiteLanguages">
+  <span ng-repeat="(language['id'], language['text']) for language in supportedSiteLanguages">
```

Comment:
This doesn't seem to match `langCode` and `langName` as they were used previously. 

Actually, this entire file is not being used anywhere, and can be removed.